frappe/automation/doctype/assignment_rule/test_assignment_rule.py

```diff
@@ -335,7 +384,7 @@ def _make_test_record(**kwargs):
 
 def create_test_doctype(doctype: str):
 	"""Create custom doctype."""
-	frappe.db.delete("DocType", doctype)
+	frappe.delete_doc("DocType", doctype)
```

Comment:
I changed this because `db.delete` doesn't affect child table rows, so multiple execution (during dev) would result in duplicate field definitions.